docs/source/building/building.rst

```diff
@@ -76,6 +80,11 @@ The dependencies can be installed via the package manager
 
 (in new terminals, re-activate the environment with ``spack env activate hipace-dev`` again)
 
+.. note::
+   On Ubuntu distributions, the InstallErrror ``"OpenMPI requires both C and Fortran compilers"`` can occur because the Fortran compilers are sometimes not set automatically in Spack.
```

Comment:
```suggestion
   On Ubuntu distributions, the InstallError ``"OpenMPI requires both C and Fortran compilers"`` can occur because the Fortran compilers are sometimes not set automatically in Spack.
```